### DISCUSSION TITLE
Any way to play around with it without cloning the repo?

### DISCUSSION THREAD

**Discussion by rumanHuq** [vue-vine/vue-vine#28] [Q&A] [answered]
Hey, Tremendous work! When would it be possible to access through `npm install vue-vine` at some point? Also, the vscode extension is a superb addition as well. Hope it is also accessible from vscode marketplace someday 😇

**Comment by baiwusanyu-c** [comment, marked as answer]
Thank you for your attention
see：https://github.com/vue-vine/vue-vine/issues/17

**Comment by ShenQingchuan** [comment]
To @rumanbsl 

Thank you for your comments. As the initiator of the project, I would like to elaborate on the current plan in more detail.

We plan to launch the first release after we've finished the VSCode extension development, it's supposed to provide:
1. Syntax highlighting for VineComponentFunction
2. CSS (and also includes pre-processors) intellisense inside `vineStyle` call
3. Vue template intellisense inside `vine` tagged template string

For now, we're holding up for Volar.js to support TS Plugin integration, in order to leverage more convenience it provides for supporting TypeScript features inside Vue template, since Vine's file structure and code organization have a lot of difference with SFC, we can't re-use existing solution.

Beyond that, we're planning to release a online playground on our documentation site, which you can see the compiler's transform result, just like [Vuejs Official SFC Playground](https://play.vuejs.org)

We're pretty exicited about what we build for now, but we know we still have a long way to go. If you have any experience about VSCode embbed language extension development, or you're interested in any part of this project, you can definetely join us!

Any suggestions would be appreciated. Thanks again.

**Reply by rumanHuq** [reply]
> To @rumanbsl
> 
> Thank you for your comments. As the initiator of the project, I would like to elaborate on the current plan in more detail.
> 
> We plan to launch the first release after we've finished the VSCode extension development, it's supposed to provide:
> 
>     1. Syntax highlighting for VineComponentFunction
> 
>     2. CSS (and also includes pre-processors) intellisense inside `vineStyle` call
> 
>     3. Vue template intellisense inside `vine` tagged template string
> 
> 
> For now, we're holding up for Volar.js to support TS Plugin integration, in order to leverage more convenience it provides for supporting TypeScript features inside Vue template, since Vine's file structure and code organization have a lot of difference with SFC, we can't re-use existing solution.
> 
> Beyond that, we're planning to release a online playground on our documentation site, which you can see the compiler's transform result, just like [Vuejs Official SFC Playground](https://play.vuejs.org)
> 
> We're pretty exicited about what we build for now, but we know we still have a long way to go. If you have any experience about VSCode embbed language extension development, or you're interested in any part of this project, you can definetely join us!
> 
> Any suggestions would be appreciated. Thanks again.

Appreciate your extensive reply. Given I am an application developer, and I can help in writing documentation, recording video tutorials etc. 

I have a couple of questions. If you can shed some light:
- would `vineStyle` accept props similar to styled-components?
```tsx
const pStyles = vineStyle(props => `
  p {
    color: ${props.foo};
  }
`)
```
- would `css-module` support be provided?
- How would it play with SSR framework like NuxtJS?

**Comment by rumanHuq** [comment]
One further suggestion is, if you can provide a Roadmap for better visibility :)
